pkg/scheduler/reconciliation/persistent.go

```diff
@@ -551,7 +551,8 @@ func (r *PersistentReconciliationRepository) GetComponentOperationProcessingDura
 		return 0, err
 	}
 	if len(operations) == 0 {
-		return 0, errors.Errorf("No operation for component %s found with desired state %s", component, state)
+		r.Logger.Infof("No operation for component %s found with desired state %s", component, state)
```

Comment:
Not sure if even `Info` is too much.. do we have to know it as it's important to trace things or is it just interesting when a developer is looking on it?

Maybe `Debug` could be better suited

Reply:
Changed